sdks/go/pkg/beam/transforms/stats/stats.shims.go

```diff
@@ -68,31 +68,31 @@ func init() {
 	runtime.RegisterType(reflect.TypeOf((*meanAccum)(nil)).Elem())
```

Comment:
I believe this file should not be present in change.
It seem to be formatting change only though.